modules/core/04-channel/v2/keeper/packet.go

```diff
@@ -53,15 +54,17 @@ func (k *Keeper) sendPacket(
 		return 0, "", errorsmod.Wrapf(clienttypes.ErrInvalidHeight, "cannot send packet using client (%s) with zero height", sourceClient)
 	}
 
+	// client timestamps are in nanoseconds while packet timeouts are in seconds
+	// thus to compare them, we convert the client timestamp to seconds in uint64
+	// to be consistent with IBC V2 specified timeout behaviour
 	latestTimestamp, err := k.ClientKeeper.GetClientTimestampAtHeight(ctx, sourceClient, latestHeight)
```

Comment:
nit: would be slightly easier to read the code below if this was called latestTimestampNano (and would avoid accidental usage later)